spdmlib/src/responder/measurement_rsp.rs

```diff
@@ -518,47 +518,6 @@ mod tests_responder {
                 assert_eq!(payload.number_of_measurement, 1);
                 assert_eq!(payload.slot_id, 0);
                 assert_eq!(payload.measurement_record.number_of_blocks, 1);
-                //index in measurement_record should equal to measurement_operation
```

Comment:
why remove the check?

Reply:
> why remove the check?

Because the record field of SpdmMeasurementRecordStructure is removed in previous commit, it is not available any more.

https://github.com/jyao1/rust-spdm/commit/d9aa6ca58d9506e1f9b817be4a2e51418c91b5e3#diff-0d8e658ce73b71f92ff8085b92cad0c4281cab172f308d0f05e328f934454f6eR963-R966